app/models/course.rb

```diff
@@ -55,4 +55,8 @@ def both_study_modes_available?
   def full?
     course_options.all?(&:no_vacancies?)
   end
+
+  def find_url
+    @find_url ||= "https://www.find-postgraduate-teacher-training.service.gov.uk/course/#{provider.code}/#{code}"
```

Comment:
Is the memoization necessary?

```suggestion
    "https://www.find-postgraduate-teacher-training.service.gov.uk/course/#{provider.code}/#{code}"
```